change-notes/1.23/analysis-javascript.md

```diff
@@ -38,6 +38,7 @@
 | Reflected cross-site scripting (`js/reflected-xss`) | Fewer false-positive results | The query now recognizes more sanitizers. |
 | Stored cross-site scripting (`js/stored-xss`) | Fewer false-positive results | The query now recognizes more sanitizers. |
 | Uncontrolled command line (`js/command-line-injection`) | More results | This query now treats responses from servers as untrusted. |
+| Loop bound injection (`js/loop-bound-injection`) | Fewer false-positive results | This query now handles when the index variable in a `_.forEach(..)` loop is used. This query is now shown by default on LGTM. |
```

Comment:
This query is already mentioned in the "new queries" section. Just modify that entry instead.